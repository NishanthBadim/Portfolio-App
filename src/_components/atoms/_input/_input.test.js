import React from "react";
import { create } from "react-test-renderer";
import { render, screen, getByRole } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";
import InputBox from "./_input";

describe("Input Box testing", () => {
  test("Matches the snapshot", () => {
    const input = create(<InputBox />);
    expect(input.toJSON()).toMatchSnapshot();
  });

  test("Input field is present", () => {
    const {getByRole} = render(<InputBox />);
    const input = getByRole("textbox");
    expect(input).toBeInTheDocument();
  });


});