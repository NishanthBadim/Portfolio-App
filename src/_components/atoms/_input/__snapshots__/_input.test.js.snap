// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input Box testing Matches the snapshot 1`] = `
<div>
  <div
    className="MuiInputBase-root MuiInput-root MuiInput-underline makeStyles-input-1 MuiInputBase-fullWidth MuiInput-fullWidth"
    onClick={[Function]}
  >
    <input
      className="MuiInputBase-input MuiInput-input"
      onAnimationStart={[Function]}
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      type="text"
    />
  </div>
</div>
`;
